stubs/jwcrypto/jwcrypto/jwk.pyi

```diff
@@ -29,10 +29,10 @@ priv_bytes: Incomplete
 JWKTypesRegistry: Incomplete
 
 class ParmType(Enum):
-    name: str
-    b64: str
-    b64u: str
-    unsupported: str
+    name = "A string with a name"
```

Comment:
I think this will need an ignore to silence pyright. (Pyright is correct about the error, but nothing we can do about it here.)